### DISCUSSION TITLE
DietPi-Software | Castopod

### DISCUSSION THREAD

**Discussion by itsdoublearon** [MichaIng/DietPi#6730] [Software Request]
Hi there, I would like to request Castopod please be added.
https://castopod.org/

**Comment by MichaIng** [comment]
Interesting, this is something new, I mean we have a lot of video/audio steaming servers, but not for podcasts in particular.

Code via GitLab: https://code.castopod.org/adaures/castopod
GitHub mirror: https://github.com/ad-aures/castopod
Install instructions: https://docs.castopod.org/getting-started/install.html

A classic PHP web application.
